_includes/api/en/4x/res-cookie.md

```diff
@@ -13,6 +13,7 @@ The `options` parameter is an object that can have the following properties.
 | `path`      | String | Path for the cookie. Defaults to "/".
 | `secure`    | Boolean | Marks the cookie to be used with HTTPS only.
 | `signed`    | Boolean | Indicates if the cookie should be signed.
+| `encode`    | Function | A synchronous function used for cookie value encoding.
```

Comment:
Hi! The rest of this table is in alphabetical order. Please feel free to update your pull request to add the option in the correct order :)